app/src/main/java/fr/free/nrw/commons/feedback/Feedback.java

```diff
@@ -0,0 +1,171 @@
+package fr.free.nrw.commons.feedback;
+
+/**
+ * Pojo class for storing information that are required while uploading a feedback
+ */
+public class Feedback {
+    // Version of app
+    private String version;
+    // API level of user's phone
+    private String apiLevel;
+    // Title/Description entered by user
+    private String title;
+    // Android version of user's device
+    private String androidVersion;
+    // Device Model of user's device
+    private String deviceModel;
+    // Device manufacturer name
+    private String deviceManufacturer;
+    // Device name stored on user's device
+    private String device;
+    // network type user is having (Ex: Wifi)
+    private String networkType;
+
+    public Feedback(final String version, final String apiLevel, final String title, final String androidVersion,
+        final String deviceModel, final String deviceManufacturer, final String device, final String networkType
+        ) {
+        this.version = version;
+        this.apiLevel = apiLevel;
+        this.title = title;
+        this.androidVersion = androidVersion;
+        this.deviceModel = deviceModel;
+        this.deviceManufacturer = deviceManufacturer;
+        this.device = device;
+        this.networkType = networkType;
+    }
+
+    /**
+     * Get the version from which this piece of feedback is being sent.
+     * Ex: 3.0.1
+     * @return
```

Comment:
Could you please remove the empty @return line here and below?
And also please remove lines such as `@param version`.
These lines are only useful if you add some explanation at the end of the line.
Thanks!